<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metis Programmer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>801</width>
      <height>91</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="programmerTab">
     <attribute name="title">
      <string>Metis Programmer</string>
     </attribute>
     <widget class="QPushButton" name="discoverPushButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Discover</string>
      </property>
     </widget>
     <widget class="QComboBox" name="metisComboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>571</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Metis:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Metis:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bootloaderTab">
     <attribute name="title">
      <string>Metis Boot Loader</string>
     </attribute>
     <widget class="QLabel" name="privilegesLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>You must be running with Administrator privileges to be able to read/write raw ethernet frames.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>781</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Check that there is a jumper on JP1 of Metis and that it has been power cycled.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="programTab">
     <attribute name="title">
      <string>Program</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Program File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileLineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>541</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="browsePushButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="programPushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Program</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="eraseTab">
     <attribute name="title">
      <string>Erase</string>
     </attribute>
     <widget class="QPushButton" name="erasePushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Erase</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ipAddressTab">
     <attribute name="title">
      <string>IP Address</string>
     </attribute>
     <widget class="QPushButton" name="readIPPushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QPushButton" name="writeIPPushButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ipALineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ipBLineEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ipCLineEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ipDLineEdit">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="macAddressTab">
     <attribute name="title">
      <string>MAC Address</string>
     </attribute>
     <widget class="QPushButton" name="readMACPushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="macLineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>281</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="statusListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>801</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Interface</string>
    </property>
    <widget class="QComboBox" name="interfaceComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="interfaceLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>731</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
